waterloop-site/src/components/SustainableTech/Hero/Hero.tsx

```diff
@@ -1,5 +1,93 @@
 import React from 'react';
+import styled from 'styled-components';
 
-const Hero: React.FC = () => <div>Hero</div>;
+import LandingImage from '../../../static/img/sustainable-tech/HeroLandingImage.svg';
+import GreenTechLogo from '../../../static/img/sustainable-tech/GreenTechLogo.svg';
 
+const LandingContainer = styled.div`
+  padding: 60px 0 30px;
```

Comment:
Is this required? When I run the branch I see an empty blue space at the bottom, and removing this padding fixes that.